python/flashinfer/gemm.py

```diff
@@ -65,6 +67,172 @@ def get_gemm_sm90_module():
             )
     return _gemm_module_sm90
 
+@triton.jit
+def compute_sm80_group_gemm_args(
+    all_problems_ptr,
+    x_ptr, w_ptr, y_ptr,
+    x_ld_ptr, w_ld_ptr, y_ld_ptr,
+    x, w, y,
+    xy_indptr, w_indices,
+    d_in, d_out, w_column_major
+):
+    
+    pid = tl.program_id(0)
+
+    m = tl.load(xy_indptr + pid + 1) - tl.load(xy_indptr + pid)
+    k, n = d_in, d_out
+
+    tl.store(all_problems_ptr + pid * 3, m)
+    tl.store(all_problems_ptr + pid * 3 + 1, n)
+    tl.store(all_problems_ptr + pid * 3 + 2, k)
+
+    w_i = tl.load(w_indices + pid) if w_indices else pid
+    w_curr_ptr = w + w_i * k * n
+    tl.store(w_ptr + pid, w_curr_ptr)
+
+    x_curr_ptr = x + tl.load(xy_indptr + pid) * k
+    tl.store(x_ptr + pid, x_curr_ptr)
+
+    y_curr_ptr = y + tl.load(xy_indptr + pid) * n
+    tl.store(y_ptr + pid, y_curr_ptr)
+
+    tl.store(x_ld_ptr + pid, k)
+    tl.store(w_ld_ptr + pid, k if w_column_major else n)
+    tl.store(y_ld_ptr + pid, n)
+
+@triton.jit
+def compute_sm90_group_gemm_args(
+    all_problems_ptr, 
+    x_ptr, w_ptr, y_ptr, 
+    x_stride_ptr, w_stride_ptr, y_stride_ptr, 
+    x, w, y,
+    x_strides, w_strides,
+    xy_indptr, w_indices, 
+    d_in, d_out, w_column_major):
+
+    pid = tl.program_id(0)
+    
+    m = tl.load(xy_indptr + pid + 1) - tl.load(xy_indptr + pid)
+    k, n = d_in, d_out
+
+    tl.store(all_problems_ptr + pid * 3, m) 
+    tl.store(all_problems_ptr + pid * 3 + 1, n)
+    tl.store(all_problems_ptr + pid * 3 + 2, k)
+
+    w_i = tl.load(w_indices + pid) if w_indices else pid
+    w_curr_ptr = w + w_i * k * n
+    tl.store(w_ptr + pid, w_curr_ptr)
+
+    x_curr_ptr = x + tl.load(xy_indptr + pid) * k
+    tl.store(x_ptr + pid, x_curr_ptr)
+
+    y_curr_ptr = y + tl.load(xy_indptr + pid) * n
+    tl.store(y_ptr + pid, y_curr_ptr)
+
+    if not x_strides:
```

Comment:
What's the case that `x_strides` will be None?

Reply:
`x_strides` and `weight_strides` are optional, so when user don't specify them we generate strides like using `cutlass::make_cute_packed_stride`. (is it the desired behavior?)